zgw/zaken-api/src/main/resources/config/liquibase/changelog/20250312_zaak_type_link_to_case_definition.xml

```diff
@@ -19,9 +19,16 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
 
-    <changeSet id="1" author="Ivo Zaal">
```

Comment:
Looks like you have overwritten an existing changeset, not sure if intended. If it was intended it should probably use a different id so it won't cause problems.

Reply:
If you look at this, it says I changed a file from openzaak, but this module is removed. Seems like git quirk to me, can you verify? I think I just added a new changelog file

Reply:
I think you're right. Sometimes git tried to guess if files moved. This seems to be one of those cases.